R/moma-R6.R

```diff
@@ -0,0 +1,109 @@
+#' R6 objects for storing and accessing the results of SFPCA / SFLDA /
+#' SFCCA
+#'
+#' During initialization of an \code{SFPCA} object, \code{R}
+#' calls the \code{C++}-side function, \code{cpp_multirank_BIC_grid_search}, and
+#' wraps the results returned. The \code{SFPCA} object also records penalty levels
+#' and selection schemes of tuning parameters. Several helper
+#' methods are provivded to facilitate access to results.
+#' Initialization is delegated to \code{\link{moma_sfpca}}.
+#'
+#' The following table lists the supported methods for
+#' R6 objects generated by \code{moma_*pca}, \code{moma_*cca}
+#' and \code{moma_*lda} types functions.
+#' \tabular{lccccccc}{
```

Comment:
I believe `roxygen2` supports Markdown syntax as well. That might be an easier way to make a table.

Reply:
Markdown syntax for tables don't work, even if I turn on markdown support for a package.